app.organicmaps.desktop.yml

```diff
@@ -0,0 +1,39 @@
+app-id: app.organicmaps.desktop
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: OMaps-wrapper
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: organicmaps
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - 'install -Dm755 ../wrapper.sh ${FLATPAK_DEST}/bin/OMaps-wrapper'
+    cleanup:
+      - '/lib' # static libraries and build configs
+      - '/include'
+    sources:
+      - type: git
+        # TODO set to upstream when the pr is merged
+        url: https://github.com/fgaz/organicmaps
+        # TODO on next release, add tag: YYYY.MM.DD-N-android
+        commit: fbebe70c7a4a67128cd94c6d35f29c7d65e34361
+      - type: shell
+        commands:
+          - 'bash configure.sh'
+      # Tell the program that the read-only and the read-write data locations
+      # are different, and create the read-write one.
+      - type: script
+        commands:
+          - 'data_path="${XDG_DATA_HOME:-${HOME}/.var/app/app.organicmaps.OrganicMaps/data}/OMaps"'
+          - 'mkdir -p "$data_path"'
+          - '/bin/OMaps -resources_path /share/organicmaps/data -data_path "$data_path"'
```

Comment:
there will be no `/bin/OMaps`. Nor will there be a `/share`. The install prefix is `/app`.